IRescue/Core/Utils/AbRelPositioning.cs

```diff
@@ -36,6 +36,25 @@ public static Pose GetLocation(Pose absoluteLocation, Pose relativeLocation)
         }
 
         /// <summary>
+        ///   Calculates the 2D absolute location of relatePosition with an absolute maker location and the relative location to it.
+        /// </summary>
+        /// <param name="absoluteLocation">The known absolute pose.</param>
+        /// <param name="relativeLocation">The pose relative to the absolute location.</param>
+        /// <returns>The location of the calculated absolute position.</returns>
+        public static Pose GetLocation2D(Pose absoluteLocation, Pose relativeLocation)
+        {
+            Vector3 absolutePosition = absoluteLocation.Position;
+            float distanceXY = (float)Math.Sqrt(relativeLocation.Position.X * relativeLocation.Position.X + relativeLocation.Position.Y + relativeLocation.Position.Y);
```

Comment:
Shouldnt this be `relativeLocation.Position.Y * relativeLocation.Position.Y` not `relativeLocation.Position.Y + relativeLocation.Position.Y` ?